code/obj/item/book.dm

```diff
@@ -3285,3 +3285,63 @@ soon the light of the unwaking will rise and the shining ones will not be prepar
 		if (voidMessage)
 			boutput(wearer, "[voidMessage]")
 		return
+
+/obj/item/paper/book/custom/example_1
+	name = "testing 3"
+	desc = "afnsanf"
+	book_cover = "redbook"
+	ink_color = "#FE2200"
+	info = {"this text is red"}
+
+/obj/item/paper/book/custom/example_2
+	name = "testing 4"
+	desc = "screaaaaaaaaaaaaam"
+	custom_cover = 1
+	cover_color = "#5566FF"
+	cover_symbol = "duck"
+	cover_flair = "spinecover"
+	flair_color = "#FF6655"
+	flair_colorable = 1
+	info = {"yo bitch im a fuckin duck"}
```

Comment:
Probably shouldn't be here, please remove.
```suggestion
```

Reply:
i left those in as an example, but it should be fine just to remove them

Reply:
removed these